src/components/Badge/story.js

```diff
@@ -46,10 +46,8 @@ storiesOf('Planets/Badge', module)
   .add(
     'Overview',
     () => {
-      const colorOptions = variants;
-
       const fontSize = select('Font size', fontSizes, 'sm');
-      const variant = select('Variant', colorOptions, '');
+      const variant = select('Variant', variants, '');
```

Comment:
Small thing, but I think it makes more sense to label the knobs with the precise prop name, so doc readers don't have to translate to camelcase in their heads.

Reply:
Ah, fair enough! Hadn't considered that. I can update it